src/components/InviteOptionsModal.tsx

```diff
@@ -63,13 +63,15 @@ const InviteOptionsModal = ({ recipient, onClose }: Props) => {
     </SafeAreaView>
   )
 }
+const { height, width } = Dimensions.get('window')
 
 const styles = StyleSheet.create({
   container: {
-    height: variables.height,
```

Comment:
not sure why `variables.height` didn't work well here, from observation it looks like the `variables.height` value also accounted for a notch when the device doesn't have one so it was too short :/ the only other place that uses `variables.height` is the FullscreenCTA but the way that is used is different (not inside absolutely positioned containers)